Bottom: 3c315a033698e62be0d00ca2a1cb4bc613804dc0
Top:    5089d6267ebf61d6f213ecaf0a3ddfc842e48002
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 21:05:03 +0100

Refresh of shiny-3d-eye-candy

---

diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index e59557f..7db4772 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -221,7 +221,6 @@ ghid_pinout_preview_expose (GtkWidget * widget, GdkEventExpose * ev)
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
-  glStencilMask (~0);
   glClear (GL_COLOR_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
 
   hidgl_reset_stencil_usage ();
